packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -428,6 +430,34 @@ export const HelpCenterContactForm = () => {
 		return isSubmitting ? formTitles.buttonSubmittingLabel : formTitles.buttonLabel;
 	};
 
+	// TODO: Figure out in which environments this makes sense.
+	const showGPTResponse = true;
+	if ( showGPTResponse && showingSibylResults ) {
+		return (
+			<div className="help-center__sibyl-articles-page">
+				<BackButton />
+				<HelpCenterGPT message={ message } setLoadingState={ setGPTFetching } />
+				<section className="contact-form-submit">
+					<Button
+						disabled={ gptFetching ? true : isCTADisabled() }
```

Comment:
Is this the same as this?
```suggestion
						disabled={ gptFetching || isCTADisabled() }
```